config/ola.m4

```diff
@@ -100,9 +100,30 @@ AM_CONDITIONAL(BUILD_OLA_PROTOC_PLUGIN, test "${with_ola_protoc_plugin}" == "no"
 ])
 
 
-# PLUGIN_SUPPORT(plugin, conditional, prerequisites_found)
-# Build the specified plugin, unless it was disabled at configure time.
-# -----------------------------------------------------------------------------
+# PLUGIN_SUPPORT(plugin_key, conditional, prerequisites_found)
+#
+# Build the specified plugin if requested and its dependencies are met.
+#
+# Each plugin specified with PLUGIN_SUPPORT gets its own configure switch named
+# --enable_${plugin_key}. The three acceptable values the user can give are:
```

Comment:
We should encourage users to use --disable-${plugin_key} (or --enable-${plugin_key} in conjunction with --disable-all-plugins rather than passing options into a single option. Also it's hyphens not underscores.